### PR TITLE
🐛 Ignore CRD not installed error when dry-run

### PR DESCRIPTION
**Pull request by zhujian7**
[comment]: # ( Copyright Contributors to the Open Cluster Management project )

## Related issue(s)
https://github.com/open-cluster-management-io/clusteradm/issues/416

Fixes #

### CODE REVIEW

File: pkg/helpers/reader/reader.go [hunk 1]
```diff
@@ -60,6 +61,12 @@ func (r *ResourceReader) Apply(fs embed.FS, config interface{}, files ...string)
 		Stream(bytes.NewReader(rawObjects), "local").
 		Flatten().
 		Do()
+	if r.dryRun {
+		// Ignore CRD not installed error when dry-run
+		rb.IgnoreErrors(func(err error) bool {
+			return err != nil && strings.Contains(err.Error(), "ensure CRDs are installed first")
```
Comment (qiujian16) [inline pkg/helpers/reader/reader.go:67]: just curious, do we have a certain error type for this?

Comment (zhujian7) [reply pkg/helpers/reader/reader.go:67]: seems no

**Pull request merged**